.kitchen.yml

```diff
@@ -0,0 +1,53 @@
+---
+driver:
+  name: docker
+
+driver_config:
+  use_sudo: false
+  privileged: true
+  provision_command: mkdir -p /run/sshd
+
+platforms:
+  - name: debian-9
+  - name: ubuntu-18.04
+  - name: fedora-27
+  - name: centos-7
+
+provisioner:
+  name: salt_solo
+  log_level: info
+  require_chef: false
+  salt_install: bootstrap
+  salt_version: latest
+  salt_bootstrap_url: https://bootstrap.saltstack.com
+  salt_bootstrap_options: -X -p git -p curl -p sudo
+  is_file_root: true
```

Comment:
You need to remove this line. It defines the root path for this formula, and it's at odds with the path defined in `state_top` below (that's why it's not finding the states).